snuba/clusters/cluster.py

```diff
@@ -10,6 +13,14 @@
 from snuba.writer import BatchWriter, WriterTableRow
 
 
+@dataclass(frozen=True)
+class ClickhouseHost:
```

Comment:
Aren't we supposed to distinguish between storage nodes and query nodes?
- is such parameter supposed to show up here?
- how would it be populated? Are we getting it from clickhosue or from the config, and if it comes from the config, how does that match with the configuration loaded from clickhouse ?

Reply:
Renamed to make it clearer that we are referring to the storage nodes here. This comes from either `system.clusters` in a multi node cluster, and is the same as the provided host and port (the query node) in a single node cluster.

Reply:
Ok, but I am not sure I get where we would configure the query nodes.  Are they going to be part of the cluster? The reader should come from one of the query nodes.